pbs/custom.py

```diff
@@ -42,8 +43,19 @@
 SAMPLEDIR=/media/VD_Research/NGSData/ReadsBySample/${SAMPLENAME}
 TMPDIR=$(pwd)/tmp runsample.py $SAMPLEDIR {REFPATH} {SAMPLENAME} -od {SAMPLENAME}
 '''
+expand_path = compose(os.path.realpath, os.path.expanduser) 
```

Comment:
You should also do os.path.expandvars that way they can use variables too

I think that makes sense